pyproject.toml

```diff
@@ -21,15 +21,17 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
     "Topic :: Software Development :: Compilers",
 ]
-dependencies = [
-    "llvmlite >=0.39,<0.43", "setuptools"
-]
+dependencies = ["llvmlite >=0.39,<0.43", "setuptools"]
```

Comment:
this is in conflict with the llvmlite version spec in the conda recipe above.

Reply:
PIXIE now needs an llvmlite from the `0.44.0dev0` series as llvmlite has had features added recently to specifically help out PIXIE. I presume this can be "spelled" here as `"llvmlite >=0.44.0dev0"` and the `llvmlite` package come from the Numba conda channel/a local build?

Reply:
fixed in https://github.com/numba/pixie/pull/28/commits/cb525a90cc0b5003a72e3b4f68d204492a850a0f